packages/rum-core/src/common/apm-server.js

```diff
@@ -65,8 +65,8 @@ class ApmServer {
       name: sanitizeString(cfg.get('serviceName'), stringLimit, true),
       version: sanitizeString(cfg.get('serviceVersion'), stringLimit),
       agent: {
-        name: sanitizeString(cfg.get('agentName'), stringLimit),
-        version: sanitizeString(cfg.get('agentVersion'), stringLimit)
+        name: 'js-base',
+        version: '4.0.1'
```

Comment:
The reason for using this instead of 
```js
const {version} = require('../../package.json')
```
is because the package.json would simply pulled in to the bundle because tree shaking wont work properly in CJS modules. We can switch to that once we migrate to ES6 modules.